src/charm.py

```diff
@@ -140,7 +260,7 @@ def _on_nrf_broken(self, event: EventBase) -> None:
         Args:
             event (NRFBrokenEvent): Juju event
         """
-        self.unit.status = BlockedStatus("Waiting for fiveg_nrf relation")
+        logger.info("Waiting for fiveg_nrf relation")
```

Comment:
Do we need this method at all? Can we just run the main callback method which will put the charm in the right status?

Reply:
Removed this method.